_targets.R

```diff
@@ -7,9 +7,36 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
+  tar_target(
+    site_01427207,
+    download_nwis_data("01427207"),
+    format = 'file'
+  ),
+  tar_target(
+    site_01432160,
+    download_nwis_data("01432160"),
+    format = 'file'
+  ),
+  tar_target(
+    site_01435000,
+    download_nwis_data("01435000"),
+    format = 'file'
+  ),
+  tar_target(
+    site_01436690,
+    download_nwis_data("01436690"),
+    format = 'file'
+  ),
+  tar_target(
+    site_01466500,
+    download_nwis_data("01466500"),
+    format = 'file'
+  ),
+
   tar_target(
     site_data,
-    download_nwis_data(),
+    c(site_01427207, site_01432160, site_01435000, site_01436690,site_01466500) %>%
+      map_dfr(read_csv, col_types = 'ccTdcc')
   ),
```

Comment:
Is this ok, or do we have  a policy re: just incorporating processes into the targets vs explicitly creating a function?